packages/std/Cargo.toml

```diff
@@ -30,6 +33,7 @@ stargate = []
 [dependencies]
 base64 = "0.13.0"
 cosmwasm-derive = { path = "../derive", version = "0.14.0-0.4.0" }
```

Comment:
For all about the remain old versions `0.14.0-0.4.0`,
How about changing to `0.16.0` in this PR and creating a '0.16.0-0.1.0' bump-up PR after merged this PR?
https://github.com/line/cosmwasm/pull/130

In the previous migration PR, it seems follow in the above way.

Reply:
I will remain the versions and change them in a PR once. Why do we need to change to `0.16.3` in this PR?

Reply:
It doesn't mean much. Because that's what we've done in previous works.

I think In a very small sense, the baseline version becomes clearer in the commit at the time this PR was squash merged.